introduction.md

```diff
@@ -3,4 +3,5 @@ My favourite path is JavaScript
 * ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?